FrontEnd/src/components/SignUp/components/signup/signup-form/RulesModal.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './RulesModal.module.css';
+import rulesText from './Rules_text';
+
+const RulesModal = ({ closeModal }) => {
+  return (
+    <div className={styles.modal}>
+      <div className={styles['rules-modal']}>
+        <h2 className={styles['rules-modal__title']}>{rulesText.title}</h2>
+        {rulesText.content.map((item, index) => {
+          if (item.type === 'paragraph') {
```

Comment:
not: you may keep it easier using ternary operator

Reply:
"Here is an unknown text and its design, so far I don't see the content in the ternary representation of the code. &&, the code is small and easy to read